openapi-validator/build.gradle

```diff
@@ -134,29 +142,6 @@ test {
     }
 }
 
-//publishing {
-//    publications {
-//        mavenJava(MavenPublication) {
-//            groupId project.group
-//            artifactId "openapi-validator"
-//            version = project.version
-//            artifact jar
-//        }
-//    }
-//
-//    repositories {
-//        maven {
-//            name = "GitHubPackages"
-//            url = uri("https://github.com/ballerina-platform/ballerina-openapi")
-//            credentials {
-//                username = System.getenv("packageUser")
-//                password = System.getenv("packagePAT")
-//            }
-//        }
-//    }
-//}
```

Comment:
Do we need to keep these commented lines?

Reply:
No these are related to the previous code structure and now we suppose to publish at the `module-ballerina-openapi` and therefore removed those from here.